cmake/Dependencies.cmake

```diff
@@ -70,11 +69,12 @@ endif()
 if(USE_NNPACK)
   include("cmake/External/nnpack.cmake")
   if(NNPACK_FOUND)
-    caffe2_include_directories(${NNPACK_INCLUDE_DIRS})
-    list(APPEND Caffe2_DEPENDENCY_LIBS ${NNPACK_LIBRARIES})
     if(TARGET nnpack)
       # ---[ NNPACK is being built together with Caffe2: explicitly specify dependency
-      list(APPEND Caffe2_EXTERNAL_DEPENDENCIES nnpack)
+      list(APPEND Caffe2_DEPENDENCY_LIBS nnpack pthreadpool cpuinfo)
```

Comment:
Adding just `nnpack` alone should work. NNPACK CMakeLists properly specifies its own dependencies.